buddyboss/buddyboss-platform
# Network Search - Media

### All Submissions:


### Changes proposed in this Pull Request:

Attachment search issue fixed
Media photo & albums network search added

### How to test the changes in this Pull Request:

Go to Buddyboss -> Setting ->Search
Check media https://prnt.sc/tv33ea
Go to the front and search any media name
See result

### Connected PR in Theme
https://github.com/buddyboss/buddyboss-theme/pull/1093

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
